server/utils/format.ts

```diff
@@ -0,0 +1,24 @@
+const notNumber = (n: unknown) => typeof n !== 'number' || Number.isNaN(n)
+
+export default {
+  date(date: Date) {
+    return date
+      .toLocaleDateString('en-GB', {
+        hour: '2-digit',
+        hour12: false,
+        minute: '2-digit',
+        timeZone: 'Europe/London',
+      })
+      .replace(',', '')
+  },
+
+  thousands(integer: number) {
+    if (notNumber(integer)) return '?'
+    return Math.round(integer).toLocaleString()
```

Comment:
I wonder if given the name of the function we should be more explicit and specify the locale, e.g.

```
return Math.round(integer).toLocaleString('en-GB')
```